i18n/translations/fr/form-builder.json

```diff
@@ -884,6 +884,9 @@
       "errors": {
         "translation": "Veuillez fournir du contenu équivalent dans les deux langues officielles."
       }
+    },
+    "banner": {
```

Comment:
@anikbrazeau some new line here

Again hoping you can swap around these placeholders for FR

```
"text2": "{{month}} {{day}}, {{year}} at {{hour}}:{{minute}} EST. [FR",
```